charts/index.yaml

```diff
@@ -3,164 +3,164 @@ entries:
   azuredisk-csi-driver:
```

Comment:
These changes are no longer relevant.

Reply:
Master pulled and the issue is resolved.